cli/src/builders/bob.ts

```diff
@@ -60,6 +81,14 @@ export class BobProject {
 				rulesFile.applyRule(target);
 			}
 
+			if (subdir === bnddirRulesDir) {
+				rulesFile.applyRule({
+					...this.targets.getBinderTarget(),
+					deps: this.targets.getResolvedObjects(`SRVPGM`),
+					relativePath: `${bnddirRulesDir}/app.bnddir`,
```

Comment:
is it OK to hard code to `app.bnddir`

Reply:
Good catch. I removed it and the tests are looking good.